docs/best-practice/multiple-collections-vs-single-large-collection.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 50
+title: Multiple collections vs single large collection
```

Comment:
Use title case in file names, not sentence case.

Also, this is a long title. How did it look when you built it locally?